model/_global/delete_all_pits/operations.smithy

```diff
@@ -8,9 +8,8 @@ $version: "2"
 namespace OpenSearch
 use opensearch.openapi#vendorExtensions
 
-// TODO: Fill in API Reference URL
 @externalDocumentation(
-    "API Reference": "https://opensearch.org/docs/latest"
+    "API Reference": "https://opensearch.org/docs/latest/search-plugins/point-in-time-api/#delete-pits"
 )
 
```

Comment:
This file is missing an API path DELETE /_search/point_in_time by pitIds. Added  https://github.com/opensearch-project/opensearch-api-specification/issues/99#issuecomment-1571416401